### PR TITLE
Additonal option to setup DRF.

### PR DESCRIPTION
**Pull request by jeromecaisip**
[//]: # (Thank you for helping us out: your efforts mean great deal to the project and the community as a whole!)

[//]: # (Before you proceed:)

[//]: # (1. Make sure to add yourself to `CONTRIBUTORS.rst` through this PR provided you're contributing here for the first time)
[//]: # (2. Don't forget to update the `docs/` presuming others would benefit from a concise description of whatever that you're proposing)

## Description

[//]: # (What's it you're proposing?)
I would like to add an option to setup and bootsrap DRF.

## Rationale

[//]: # (Why does the project need that?)
This could help shorten the process of setting up API and the initial pattern could
be followed so that new devs can make their API restful.

## Use case(s) / visualization(s)

[//]: # ("Better to see something once than to hear about it a thousand times.")

### CODE REVIEW

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/api/serializers.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from rest_framework import  serializers
+from {{ cookiecutter.project_slug }}.users.models import User
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = '__all__'
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/api/serializers.py]: This looks a bit too open ended. Can we lock it down a bit?

**Comment by highpost** [comment]
I am working on adding some tests for this PR. I should be done this
evening.

--Dan

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/api/serializers.py (hunk 1 above)
Comment (jeromecaisip) [reply {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/api/serializers.py]: Hi. What does it mean open ended? Should I restrict the fields? Thanks!

**Comment by jeromecaisip** [comment]
Thank you for expressing interest on my PR. I will work on them after @highpost made test for this PR to make sure it will pass the requirements.

Thanks!

**Comment by highpost** [comment]
I sent a PR to @jeromecaisip with the test code and a few other changes.

**Comment by jeromecaisip** [comment]
Hi @highpost  can you make the pull request to a different branch, name with-drf-setup.

Thanks.

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/api/serializers.py (hunk 1 above)
Comment (browniebroke) [reply {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/api/serializers.py]: Sorry, the choice of words wasn't the best. Yes, I meant that we should restrict the fields, this is exposing too much information.

**Comment by browniebroke** [comment]
I've fixed the issues and merged in #2410. Opened an issue to get tests written in #2415.

Thanks for your contribution. 🎉

**Pull request merged**